docs/source/daml2ts/index.rst

```diff
@@ -14,62 +14,87 @@ Introduction
+
+Primitive DAML types: @daml/types
+---------------------------------
+
+To understand the TypeScript code generated by ``daml2ts``, it is helpful to keep in mind this quick review of the TypeScript equivalents of the primitive DAML types provided by @daml/types.
+
+**Interfaces**:
+
+- ``Template<τ extends object, κ = unknown>``
+- ``Choice<τ extends object, ɣ, ρ, κ = unknown>``
 
 **Types**:
 
-+-------------------+------------------+----------------------------------+
-| TypeScript        | DAML equiv.      | TypeScript definition            |
-+===================+==================+==================================+
-| ``Unit``          | ``()``           | ``{}``                           |
-+-------------------+------------------+----------------------------------+
-| ``Bool``          | ``Bool``         | ``boolean``                      |
-+-------------------+------------------+----------------------------------+
-| ``Int``           | ``Int``          | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Decimal``       | ``Decimal``      | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Numeric``       | ``Numeric n``    | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Text``          | ``Text``         | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Time``          | ``Time``         | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Party``         | ``Party``        | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``List<T>``       | ``[t]``          | ``T[]``                          |
-+-------------------+------------------+----------------------------------+
-| ``Date``          | ``Date``         | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``ContractId<T>`` | ``ContractId t`` | ``string``                       |
-+-------------------+------------------+----------------------------------+
-| ``Optional<T>``   | ``Optional t``   | ``null | (null extends T ?``     |
-|                   |                  | ``[] | [Exclude<T, null>] : T)`` |
-+-------------------+------------------+----------------------------------+
-| ``TextMap<T>``    | ``TextMap t``    | ``{ [key: string]: T }``         |
-+-------------------+------------------+----------------------------------+
++-------------------+--------------------+----------------------------------+
+| DAML              | TypeScript        | TypeScript definition            |
```

Comment:
We should somehow make it clear that the types mentioned in the "TypeScript" column are defined in `@daml/types`.

Reply:
✔️